being-an-adie/homerooms.md

```diff
@@ -0,0 +1,22 @@
+# Homerooms
+
+This is a group of 24 students in either “Rock”, “Paper”, or “Scissors”. There is a Slack channel for each homeroom. The purpose of your homeroom is to create a “classroom-sized” setting within the cohort for all students to feel seen and be able to collaborate . Homerooms remain the same throughout the cohort.
```

Comment:
There are a few tiny and inconsequential spacing errors on a few docs. It doesn't really matter to me if you fix them before merging, but it might be nice to run the pretter.io format (there's a vs code extension) on each doc

```suggestion
This is a group of 24 students in either “Rock”, “Paper”, or “Scissors”. There is a Slack channel for each homeroom. The purpose of your homeroom is to create a “classroom-sized” setting within the cohort for all students to feel seen and be able to collaborate. Homerooms remain the same throughout the cohort.
```